pkg/common/options/options.go

```diff
@@ -33,6 +60,10 @@ func (o *AgentOptions) AddFlags(flags *pflag.FlagSet) {
 	_ = flags.MarkDeprecated("cluster-name", "use spoke-cluster-name flag")
 	flags.StringVar(&o.SpokeClusterName, "cluster-name", o.SpokeClusterName,
 		"Name of the spoke cluster.")
+	flags.StringVar(&o.HubKubeconfigDir, "hub-kubeconfig-dir", o.HubKubeconfigDir,
```

Comment:
I think this may have some confusion with hubkubeconfigfile, this dir is used to store the hubkubeconfig certs, could we rename it to hubkubeconfigcertdir

and we may allow this empty, if empty, we will find the certs in the dir of hubkubeconfigfile?

Reply:
hrm, I thought we also need kubeconfig to put in the same directory in registration agent?